test/system/500-networking.bats

```diff
@@ -224,6 +227,9 @@ load helpers
 }
 
 @test "podman network reload" {
+    if is_netavark; then
+        skip "Broken with netavark: https://github.com/containers/podman/issues/13073"
+    fi
```

Comment:
see https://github.com/containers/podman/issues/13073#issuecomment-1026021580

Reply:
Thanks for the help.  I'm going to step away from this PR for a bit (since the int. tests are failing).